.manylinux-install.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+# Kill the build if anything errors
+set -e -x
+
+# Compile wheels
+for PYBIN in /opt/python/*/bin; do
+    if [[ "${PYBIN}" == *"cp27"* ]] || \
+       [[ "${PYBIN}" == *"cp33"* ]] || \
+       [[ "${PYBIN}" == *"cp34"* ]] || \
+       [[ "${PYBIN}" == *"cp35"* ]]; 
+    then
+        "${PYBIN}/pip" install -r /io/requirements-doc.txt
+        "${PYBIN}/pip" install pytest
+        "${PYBIN}/pip" install -U cython
+        "${PYBIN}/pip" install argparse
+        "${PYBIN}/cython" /io/pycrfsuite/_pycrfsuite.pyx --cplus -a -I /io/pycrfsuite
```

Comment:
Is it possible to use https://github.com/scrapinghub/python-crfsuite/blob/master/update_cpp.sh script, in order to have a single place where we write how to rebuild the extension?